pkg/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -684,7 +686,7 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 							}
 
 							// TODO make configurable?
-							time.Sleep(5 * time.Second)
+							time.Sleep(1 * time.Second)
```

Comment:
is that the frequency in CDI?

Reply:
it is, but I actually reduced it for debugging and forgot to change back